ansible/aws.env.yml

```diff
@@ -0,0 +1,28 @@
+- name: Ensure aeternity network environments
+  connection: local
+  hosts: 127.0.0.1
+  gather_facts: false
+  vars:
+    - ec2_type: "m4.large"
+    - ec2_keypair: "aeternity"
+    - ec2_region: "eu-central-1"
+    - ec2_ami: "ami-8c076ce3" #ubuntu 16.04 with added master user, python and ae key from AWS.
+    - vpc_id: "vpc-3350e358"
+    - key_name: "ae"
+  tasks:
+  - name: aws
+    cloudformation:
+      profile: aeternity
+      stack_name: "ae"
```

Comment:
let's use more descriptive stack name like for OS: `ae-${envname}` in this case `ae-dev_aws`

Reply:
fixed